app/src/main/java/dev/msfjarvis/lobsters/ui/viewmodel/LobstersViewModel.kt

```diff
@@ -17,17 +19,21 @@ import javax.inject.Inject
 class LobstersViewModel @Inject constructor(
   private val pagingSource: LobstersPagingSource,
   private val lobstersRepository: LobstersRepository,
+  private val lobstersApi: LobstersApi,
 ) : ViewModel() {
   private val _savedPosts = MutableStateFlow<List<LobstersPost>>(emptyList())
+  private var source: LobstersPagingSource? = null
   val savedPosts = _savedPosts.asStateFlow()
-  val posts = Pager(PagingConfig(5)) {
-    pagingSource
-  }.flow
+  val posts = Pager(PagingConfig(25)) {
+    source = LobstersPagingSource(lobstersApi, lobstersRepository)
```

Comment:
why not initialize this directly?

Reply:
You mean via DI, like it was being done previously?

Reply:
yeah either way works, I just don't see the point of having it be a nullable field when all dependencies are available at class init.

Reply:
:+1: